portfolio/src/main/webapp/charts.js

```diff
@@ -0,0 +1,109 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+google.charts.load('current', {'packages':['corechart']});
+google.charts.setOnLoadCallback(drawPoll);
+
+// Map that holds all polls that can be voted on.
+const polls = new Map([
+  [
+    'catsDogs', {
+      title: 'Do you Prefer Cats or Dogs?',
+      choices: ['Cats', 'Dogs'],
+    }
+  ],
+  [
+    'iceCream', {
+      title: 'What is the Best Ice Cream Flavor?',
+      choices: [
+        'Vanilla', 
+        'Chocolate', 
+        'Cookies and Cream', 
+        'Coffee', 
+        'Chocolate Chip',
+      ],
+    }
+  ],
+  [
+    'pancakesWaffles', {
+      title: 'Do you Prefer Pancakes or Waffles?',
+      choices: ['Pancakes', 'Waffles'],
+    }
+  ],
+])
+
+/**
+ * Generate html for the options of polls to choose from.
+ */
+function getPollOptions() {
+  return Array.from(polls).map((poll) => { return `
```

Comment:
You can leave out the `{` and the `return` here. It's a little shortcut for arrow functions that do nothing but return a value. If you want to leave the brace and the return that's fine too but you do have to add a semicolon at the end in that case

Reply:
Thanks! Fix pushed.